Mlem/App/Views/Root/Tabs/Settings/AccountSettingsView.swift

```diff
@@ -14,17 +14,17 @@ struct AccountSettingsView: View {
     
     var body: some View {
         Form {
-            Section {
+            // empty section disables background
+            Section {} header: {
```

Comment:
This makes the banner view thinner than it was before - possibly `.listRowInsets` would fix that?

We also use the empty section in the header of the "About" page